client/app/types/course/material/folders.ts

```diff
@@ -15,11 +16,17 @@ export type FolderPermissions = Permissions<
 >;
 
 export type SubfolderPermissions = Permissions<
-  'canStudentUpload' | 'showSdlWarning' | 'canEdit' | 'canDelete'
+  | 'canStudentUpload'
+  | 'showSdlWarning'
+  | 'canEdit'
+  | 'canDelete'
+  | 'canManageKnowledgeBase'
 >;
 
 export type MaterialPermissions = Permissions<'canEdit' | 'canDelete'>;
 
+export type MaterialWorkflowState = 'not_chunked' | 'chunking' | 'chunked';
```

Comment:
See `SAVING_STATUS` `SYNC_STATUS` in `client/app/lib/constants/sharedConstants.ts`, should set the constant once, then recycle throughout.

Reply:
updated